web/src/App.vue

```diff
@@ -23,19 +23,21 @@
 import { onMounted, ref } from 'vue';
 
 import { dandiRest } from '@/rest';
-import store from '@/store';
+import { useDandisetStore } from '@/stores/dandiset';
```

Comment:
Is is the right usage pattern to name this function with a `use` prefix? In my mind, that signals that it's a [Vue composable](https://vuejs.org/guide/reusability/composables.html), whereas this just straight up returns a store object. If the usage convention is to prefix this application with `use` too, that's fine. Otherwise, I think it would be good to stick to the conventions.

Reply:
I had the same thought initially, but yes that is the convention https://pinia.vuejs.org/cookbook/migration-vuex.html#converting-a-single-module